g3doc/obtaining-training-data.md

```diff
@@ -0,0 +1,50 @@
+# Obtaining Training Data
+
+This document gives an overview of how to use the Gematria tooling to
+parse/process training data from multiple sources.
+
+## Parsing training data from the BHive dataset
+
+The publically available [BHive](https://github.com/ithemal/bhive) dataset
+has a decent amount of training data available in CSV format. Gematria
+provides native tooling to process these CSV files into `.tfrecord` files
+that can be used for further processing.
+
+For example, to process BHive throughput data for the Intel Skylake
```

Comment:
We should start by saying that the data set contains one CSV file per microarchitecture (and we can say that for example, the data for Skylake are in `skl.csv`.

That should make it easier to understand for the readers why we're picking a single microarchitecture for the example instead of putting everything to the same data set.